### ISSUE TITLE
Phase 2 related questions

### ISSUE THREAD

**Issue by EvelynNeverGiveUp** [Neuron-Hackathon/neuron_hackathon_2022#10]
1). Is the test data the same as the training data given to us? The reason why we ask this question is that the check result we have every 8 hours may have a great impact on our final plan.
2). Can we treat every 8 hours independently as a subproblem? If not, we do not understand why our reposition action cannot change the check result.

**Comment by Neuron-Hackathon** [comment]
1). No they are not the same.
2). Yes, you can treat them independently.

**Comment by ryanchua00** [comment]
Confirming 2) above,
If we begin reading data from 0000hrs, and shifts occur between 0 - 7 hours, at 0800hrs, are all the scooters moved to the position from scooter_round2 data ?

**Comment by Neuron-Hackathon** [comment]
yes, you assume that the scooters moved to the distribution stated at 0800

**Comment by EvelynNeverGiveUp** [comment]
So how about the e-scooter rented at 0759 and returned at 0810? Can we consider it returned successfully and no need to change battery and take back?

**Comment by Neuron-Hackathon** [comment]
Yes, you can consider the trips that start successfully will also end successfully and no need to change battery/take back.

**Comment by EvelynNeverGiveUp** [comment]
How can we submit presentations? Send to the specified e-mail address with other files?

**Comment by Neuron-Hackathon** [comment]
Yes, please send it to [hackathon@neuron.sg](mailto:hackathon@neuron.sg)
